packages/charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx

```diff
@@ -107,12 +112,14 @@ export class MultiStackedBarChartBase extends React.Component<IMultiStackedBarCh
       }
       value = (pointData / total) * 100;
       startingPoint.push(prevPosition);
-      const isSelected = this.state.legendSelected === point.legend!;
       const styles = this.props.styles;
+      let shouldHighlight = true;
+      if (this.state.isLegendHovered || this.state.isLegendSelected) {
+        shouldHighlight = this.state.selectedLegendTitle === point.legend ? true : false;
```

Comment:
Nit: unnecessary ternary operator, can just be `this.state.selectedLegendTitle === point.legend`

Reply:
removed the ternary operator.